step-02/README.md

```diff
@@ -14,9 +14,12 @@ AWS Amplify solved the authentication for developers. Let's use it.
 ### Library
 
 Install package, core library and react specific.
+
 ```
 npm install --save aws-amplify
 npm install --save aws-amplify-react
+npm install --save crypto-browserify
```

Comment:
crypto-browserify and path-to-regexp are dependencies of aws-amplify, no need to install explicitly.